package.json

```diff
@@ -88,6 +88,7 @@
     "**/minimist": "^1.2.5",
     "**/node-forge": "^1.2.1",
     "**/pdfkit/crypto-js": "4.0.0",
+    "**/react-markdown/html-to-react": "^1.4.8",
```

Comment:
The existing (transitive) dependency in the lockfile is `html-to-react@^1.3.4`, so you can remove this yarn dependency resolution, it's not needed. If you remove it and re-run `yarn`, the package version specified in the lockfile will still be `1.4.8` :+1:

```suggestion
```

Reply:
Great catch! No idea how I ended up with this resolution :man_facepalming:

Reply:
Thanks! :heart: